### PR TITLE
Semantic

### PR DESCRIPTION
**Pull request by idanstark42**

### CODE REVIEW

File: dev/mock-api-router.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ router.get('/match/current', (req, res) => {
 
 router.get('/match/upcoming/:count', (req, res) => {
   const responses = mockResponses()
-  res.send(responses.matches.slice(1, req.params.count + 1))
+  res.send(responses.matches.slice(1, 3))
```
Comment (idanstark42) [inline dev/mock-api-router.js:42]: The count here is intentional...

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
     "@first-lego-league/eslint-config": "^1.0.0",
-    "@first-lego-league/user-interface": "1.3.7",
+    "@first-lego-league/user-interface": "2.0.3",
```
Comment (idanstark42) [inline package.json]: Use roof `^2.0.3`

**Review by idanstark42 — COMMENTED** [review]
In the console there's a warning I don't recognize: `react.development.js?72d0:188 Warning: Each child in a list should have a unique "key" prop. Check the render method of `TeamsTable`. See https://fb.me/react-warning-keys for more information.`

Another thing to notice is the page layout requiring scroll (at least on my screen). It shuold be forced to fit into `100vh`. This can be accomplished by strongly controlling the styles & heights of the page. Since this is not a classic webpage, and its structure is based on horizontal division, you should control the height of the elements using percentages, and skip the usage of most of the rows and columns (i.e. make the `wrapper` a `grid` and put all the other elements in columns directly in it, and controlling their height using `#id`s.

File: dev/mock-api-router.js (hunk 1 above)
Comment (taltaub22) [reply dev/mock-api-router.js:42]: It did not work for me so I hard coded it to test

Comment (idanstark42) [reply dev/mock-api-router.js:42]: Ok. Maybe figure out why it didn't work

File: package.json (hunk 2 above)
Comment (idanstark42) [reply package.json]: Also use 2.1.2

**Pull request merged**